Gdk-3.0/Gtk-3.0 supports multiple targets now, and the "target" variable
no longer works. So, sort of hack around it for now?
---
diff --git a/configure.in b/configure.in
index 33cd3e8..2a71c95 100644
--- a/configure.in
+++ b/configure.in
@@ -412,12 +412,19 @@ esac
 
 # We have a direct dependency on X11 on gdk-x11, see bug #613525
 AC_MSG_CHECKING([for GDK target])
-GDK_TARGET="$($PKG_CONFIG --variable target gdk-$GTK_API_VERSION)"
-AC_MSG_RESULT([$GDK_TARGET])
-case "$GDK_TARGET" in
-  x11) PLATFORM_PKGS="x11 cairo-xlib" ;;
-  *) PLATFORM_PKGS="" ;;
+case "$with_gtk" in
+  2.0)	GDK_TARGETS=$($PKG_CONFIG --variable target gdk-$GTK_API_VERSION)
+  	;;
+  3.0)	GDK_TARGETS=$($PKG_CONFIG --variable targets gdk-$GTK_API_VERSION)
+  	;;
 esac
+AC_MSG_RESULT([$GDK_TARGETS])
+PLATFORM_PKGS=
+for i in $GDK_TARGETS; do
+  if test "$i" = "x11"; then
+    PLATFORM_PKGS="$PLATFORM_PKGS x11 cairo-xlib"
+  fi
+done
 
 VTE_PKGS="glib-2.0 >= $GLIB_REQUIRED gobject-2.0 pango >= $PANGO_REQUIRED gtk+-$GTK_API_VERSION >= $GTK_REQUIRED gobject-2.0 gio-2.0 gio-unix-2.0 $PLATFORM_PKGS"
 PKG_CHECK_MODULES([VTE],[$VTE_PKGS])
